### PR TITLE
feature(translation): add Spanish translation

### PR DESCRIPTION
**Pull request by khvilaboa**
It is a first translation. I probably will improve it in the next days but any review is welcomed.

Regards!

### CODE REVIEW

**Comment by lex111** [comment]
@khvilaboa you watched PR #282?

File: src/locale/languages/es.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const en = {
```
Comment (andrewda) [inline src/locale/languages/es.js]: Should be `es`

Comment (khvilaboa) [reply src/locale/languages/es.js]: changed

**Comment by khvilaboa** [comment]
@lex111 Yes, I had taken it into account.

**Comment by Rishabh04-02** [comment]
@khvilaboa This will pass all tests if you will change ``double quotes "`` to ``single quote '`` in the following lines - [L28](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L28) , [L86](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L86) , [L105](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L105) , [L108](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L108) , [L110](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L110) , [L115](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L115) , [L138](https://github.com/khvilaboa/git-point/blob/b84f8f69a28882a500bb61a176883be1e3f160f8/src/locale/languages/es.js#L138)

Suggestion based on the TravisCI [Errors Log](https://travis-ci.org/gitpoint/git-point/builds/284583105#L501)

**Comment by alejandronanez** [comment]
There's a PR already open for this feature. I'm merging it so we don't get any problems later. 👍

File: src/locale/languages/es.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+      title: 'Nueva incidencia',
+      missingTitleAlert: 'Debes especificar un título para la incidencia!',
+      issueTitle: 'Título de la incidencia',
+      writeATitle: 'Escribe un título para la incidencia aquí',
+      issueComment: 'Comentario de la incidencia',
+      writeAComment: 'Escribe un comentario para tu incidencia aquí',
+    },
+    pullMerge: {
+      title: 'Fusionar Pull Request',
+      createMergeCommit: 'Create a merge commit',
```
Comment (alejandronanez) [inline src/locale/languages/es.js]: You missed this one.

File: src/locale/languages/es.js [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+      squashAndMerge: 'Squash y Merge',
+      merge: 'fusionar',
```
Comment (alejandronanez) [inline src/locale/languages/es.js]: I think we should leave `squash` and `merge` and any git reserved word in English. What you guys think @lex111 @khvilaboa @andrewda @housseindjirdeh ?

Comment (lex111) [reply src/locale/languages/es.js]: @alejandronanez it all depends on how you decided to translate in your language these phrases..

For example, you can see how this is done in the [book on Git, the Spanish version](https://git-scm.com/book/es/v1/Ramificaciones-en-Git-Procedimientos-b%C3%A1sicos-para-ramificar-y-fusionar).

Comment (khvilaboa) [reply src/locale/languages/es.js]: @alejandronanez I commited a version with the git reserved words without translate (I have seen that other translations follow this pattern).

**Comment by khvilaboa** [comment]
@andrewda @alejandronanez What do you think of the current version?

File: src/locale/languages/es.js [hunk 4]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
+        'No hacemos nada con tu información de GitHub. Tras la autenticación, el token OAuth del usuario se guarda directamente en el almacenamiento de su dispositivo. No es posible para nosotros recuperar esa información. Nunca vemos ni almacenamos el token de acceso de un usuario.',
       userData2:
-        "This means that in no way, shape or form do we ever view, use or share a user's GitHub data. If private data ever becomes visible at any point we will not record or view it. If it happens to be accidentally recorded, we will delete it immediately using secure erase methods. Again, we've set up authentication specifically so that this never happens.",
-      analyticsInfoTitle: 'ANALYTICS INFORMATION',
+        'Esto significa que de ninguna manera miramos, usamos o compartimos datos de GitHub de los usuarios. Si la información privada se volviera visible en algún momento, no la registraremos ni leeremos. Si se guardara accidentalmente, la borraremos inmediatamente usando métodos seguros de borrado. Sin embargo, hemos configurado la autenticación para que esto nunca suceda.',
+      analyticsInfoTitle: 'INFORMACIÓN ANALÍTICA',
```
Comment (alejandronanez) [inline src/locale/languages/es.js]: I think `Información de Analytics` would be better.

File: src/locale/languages/es.js [hunk 5]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
       analyticsInfo1:
-        'We currently use Google Analytics and iTunes App Analytics to help us measure traffic and usage trends for the GitPoint. These tools collect information sent by your device including device and platform version, region and referrer. This information cannot reasonably be used to identify any particular individual user and no personal information is extracted.',
+        'Actualmente usamos Google Analytics y iTunes App Analytics para ayudarnos a medir el tráfico y las tendencias de uso de GitPoint. Estas herramientas recogen información mandada por tu dispositivo, invluyendo la versión del dispositivo y la plataforma, zona geográfica y These tools collect information sent by your device including device and platform version, region and referente. Esta información no puede ser usada para identificar ningún usuario individual y no se extrae ninguna información personal.',
```
Comment (alejandronanez) [inline src/locale/languages/es.js]: Please remove this.
`These tools collect information sent by your device including device and platform version, region and referente.`

**Comment by khvilaboa** [comment]
@alejandronanez English text removed

File: src/locale/languages/es.js (hunk 4 above)
Comment (alejandronanez) [reply src/locale/languages/es.js]: @khvilaboa can you please update this?

Comment (khvilaboa) [reply src/locale/languages/es.js]: @alejandronanez  Updated

File: src/locale/languages/es.js [hunk 6]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
-    stars: 'Stars',
-    orgs: 'ORGANIZATIONS',
-    noOrgsMessage: 'No organizations',
-    info: 'INFO',
-    company: 'Company',
-    location: 'Location',
+    stars: 'Estrellas',
+    orgs: 'ORGANIZACIONES',
+    noOrgsMessage: 'No hay organizaciones',
+    info: 'INFORMACIÖN',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: INFORMACI**Ó**N

File: src/locale/languages/es.js [hunk 7]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
       title: 'Pull Requests',
-      openButton: 'Open',
-      closedButton: 'Closed',
-      searchingMessage: 'Searching for {{query}}',
-      noOpenPulls: 'No open pull requests found!',
-      noClosedPulls: 'No closed pull requests found!',
+      openButton: 'Abiertos',
+      closedButton: 'Cerrados',
+      searchingMessage: 'Buscando {{query}}',
+      noOpenPulls: 'No se encontraron pull requests abiertos!',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

File: src/locale/languages/es.js [hunk 8]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
+      noClosedPulls: 'No se encontraron pull requests cerrados!',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

File: src/locale/languages/es.js [hunk 9]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
-      openButton: 'Open',
-      closedButton: 'Closed',
-      searchingMessage: 'Searching for {{query}}',
-      noOpenIssues: 'No open issues found!',
-      noClosedIssues: 'No closed issues found!',
+      openButton: 'Abiertas',
+      closedButton: 'Cerradas',
+      searchingMessage: 'Buscando {{query}}',
+      noOpenIssues: 'No se encontraron issues abiertas!',
+      noClosedIssues: 'No se encontraron issues cerradas!',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

File: src/locale/languages/es.js [hunk 10]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
       title: 'Issues',
-      openButton: 'Open',
-      closedButton: 'Closed',
-      searchingMessage: 'Searching for {{query}}',
-      noOpenIssues: 'No open issues found!',
-      noClosedIssues: 'No closed issues found!',
+      openButton: 'Abiertas',
+      closedButton: 'Cerradas',
+      searchingMessage: 'Buscando {{query}}',
+      noOpenIssues: 'No se encontraron issues abiertas!',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

File: src/locale/languages/es.js [hunk 11]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
-      codePushChecking: 'Checking for update...',
-      codePushUpdated: 'App is up to date',
-      codePushAvailable: 'Update is available!',
-      codePushNotApplicable: 'Not applicable in debug mode',
+      orgsRequestApprovalTop: '¿No ves todas tus organizaciones?',
+      orgsRequestApprovalBottom: 'Quizá tengas que solicitar aprobación para verlas.',
+      codePushCheck: 'Comprobar actualizaciones',
+      codePushChecking: 'Comprobando actualizaciones...',
+      codePushUpdated: 'Aplicación actualizada',
+      codePushAvailable: 'Actualizaciones disponibles!',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

File: src/locale/languages/es.js [hunk 12]
```diff
@@ -1,46 +1,48 @@
...
     welcome: {
       welcomeTitle: 'Bienvenido a GitPoint',
     },
     events: {
       welcomeMessage:
-        "Welcome! This is your news feed - it'll help you keep up with recent activity on repositories you watch and people you follow.",
-      commitCommentEvent: 'commented on commit',
-      createEvent: 'creado {{object}}',
-      deleteEvent: 'borrado {{object}}',
+        'Bienvenido! Este es tu hilo de noticias - te ayudará a mantenerte informado sobre la actividad de los respositorios y los usuarios que sigues.',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Start with `¡`.

**Comment by khvilaboa** [comment]
@ocarreterom Thanks for the review! Translation updated.

**Comment by khvilaboa** [comment]
@andrewda What do you think of the current translation?

File: src/locale/languages/es.js [hunk 13]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
-      numFilesChanged: '{{numFilesChanged}} files',
-      new: 'NEW',
-      deleted: 'DELETED',
-      fileRenamed: 'File renamed without any changes',
+      title: 'Diferencias',
+      numFilesChanged: '{{numFilesChanged}} archivos',
+      new: 'NUEVOS',
+      deleted: 'ELIMINADOS',
+      fileRenamed: 'Archivo renombrado sin cambios',
     },
```
Comment (chinesedfan) [inline src/locale/languages/es.js]: @khvilaboa Please add these fields.
```
    readMe: {
      readMeActions: 'README Actions',
      noReadMeFound: 'No README.md found',
    },
```

File: src/locale/languages/es.js [hunk 14]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
   },
   organization: {
     main: {
-      membersTitle: 'MEMBERS',
-      descriptionTitle: 'DESCRIPTION',
+      membersTitle: 'MIEMBROS',
+      descriptionTitle: 'DESCRIPCIÓN',
     },
```
Comment (chinesedfan) [inline src/locale/languages/es.js]: organizationActions: 'Organization Actions',

File: src/locale/languages/es.js [hunk 15]
```diff
@@ -49,261 +51,272 @@ export const erase = {
...
+        open: 'Abierta',
+        closed: 'Cerrada',
         merged: 'Merged',
       },
       screenTitles: {
         issue: 'Issue',
         pullRequest: 'Pull Request',
       },
+      openIssueSubTitle: '#{{number}} abiertas hace {{time}} por {{user}}',
+      closedIssueSubTitle: '#{{number}} por {{user}} fue cerrada hace {{time}}',
```
Comment (chinesedfan) [inline src/locale/languages/es.js]: issueActions: 'Issue Actions',

File: src/locale/languages/es.js [hunk 16]
```diff
@@ -318,5 +331,8 @@ export const erase = {
       y: '%dy',
       yy: '%dy',
     },
+    abbreviations: {
+      thousand: 'k',
+    },
```
Comment (chinesedfan) [inline src/locale/languages/es.js]: openInBrowser: 'Open in Browser',

**Comment by khvilaboa** [comment]
@chinesedfan Fields added. Thanks!

**Comment by chinesedfan** [comment]
@khvilaboa Good. And can you also fix those [ESLint errors](https://travis-ci.org/gitpoint/git-point/builds/288223660)? Some tabs is mixed into source codes.

```
/home/travis/build/gitpoint/git-point/src/locale/languages/es.js
  242:2  error  Unexpected tab character  no-tabs
  286:2  error  Unexpected tab character  no-tabs
  286:2  error  Mixed spaces and tabs     no-mixed-spaces-and-tabs
  287:2  error  Unexpected tab character  no-tabs
  343:2  error  Unexpected tab character  no-tabs
```

**Comment by khvilaboa** [comment]
@housseindjirdeh Conflicts resolved. Regards!

File: src/locale/languages/es.js [hunk 17]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
+      issueActions: 'Acciones de la Issue',
     },
     newIssue: {
-      title: 'New Issue',
-      missingTitleAlert: 'You need to have an issue title!',
-      issueTitle: 'Issue Title',
-      writeATitle: 'Write a title for your issue here',
-      issueComment: 'Issue Comment',
-      writeAComment: 'Write a comment for your issue here',
+      title: 'Nueva Issue',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Nueva issue

File: src/locale/languages/es.js [hunk 18]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
+      writeATitle: 'Escribe un título para la issue aquí',
+      issueComment: 'Comentario de la issue',
+      writeAComment: 'Escribe un comentario para tu issue aquí',
     },
     pullMerge: {
       title: 'Merge Pull Request',
-      createMergeCommit: 'Create a merge commit',
-      squashAndMerge: 'Squash and merge',
+      createMergeCommit: 'Crear un commit merge',
+      squashAndMerge: 'Squash y Merge',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Squash y merge

File: src/locale/languages/es.js [hunk 19]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
-      commitMessage: 'Commit Message',
-      writeAMessage: 'Write a message for your commit here',
-      mergeType: 'Merge Type',
-      changeMergeType: 'Change Merge Type',
+      missingTitleAlert: '¡Debes especificar un título para el commit!',
+      commitTitle: 'Título del commit',
+      writeATitle: 'Escribe aquí un título para tu commit',
+      commitMessage: 'Mensaje del commit',
+      writeAMessage: 'Escribe aquí un mensaje para tu commit',
+      mergeType: 'Tipo de Merge',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Tipo de merge

File: src/locale/languages/es.js [hunk 20]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
+      changeMergeType: 'Cambiar Tipo de Merge',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Cambiar tipo de merge

File: src/locale/languages/es.js [hunk 21]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
+    company: 'Compañía',
+    location: 'Ubicación',
     email: 'Email',
-    website: 'Website',
-    repositories: 'Repositories',
-    followers: 'Followers',
-    following: 'Following',
-    cancel: 'Cancel',
-    yes: 'Yes',
+    website: 'Sitio Web',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Sitio web

File: src/locale/languages/es.js [hunk 22]
```diff
@@ -228,86 +236,93 @@ export const erase = {
...
+      unlockIssue: 'Desbloquear {{issueType}}',
+      lockIssue: 'Bloquear {{issueType}}',
+      closeIssue: 'Cerrar {{issueType}}',
+      reopenIssue: 'Reabrir {{issueType}}',
+      areYouSurePrompt: '¿Estás seguro?',
+      applyLabelTitle: 'Asignar etiqueta a esta issue',
+    },
+    comment: {
+      commentActions: 'Acciones de comentarios',
+      editCommentTitle: 'Editar Comentario',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Editar comentario

File: src/locale/languages/es.js [hunk 23]
```diff
@@ -228,86 +236,93 @@ export const erase = {
   },
   organization: {
     main: {
-      membersTitle: 'MEMBERS',
-      descriptionTitle: 'DESCRIPTION',
+      membersTitle: 'MIEMBROS',
+      descriptionTitle: 'DESCRIPCIÓN',
     },
+    organizationActions: 'Organization Actions',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Needs translate.

File: src/locale/languages/es.js [hunk 24]
```diff
@@ -49,177 +51,183 @@ export const erase = {
...
+      readMe: 'LEEME',
+      viewSource: 'Ver Código',
       issuesTitle: 'ISSUES',
-      noIssuesMessage: 'No issues',
-      noOpenIssuesMessage: 'No open issues',
-      viewAllButton: 'View All',
-      newIssueButton: 'New Issue',
+      noIssuesMessage: 'No hay issues',
+      noOpenIssuesMessage: 'No hay issues abiertas',
+      viewAllButton: 'Ver Todo',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Ver todo

File: src/locale/languages/es.js [hunk 25]
```diff
@@ -49,177 +51,183 @@ export const erase = {
...
+      unstarAction: 'Desmarcar Favorito',
+      shareAction: 'Compartir',
+      unwatchAction: 'Dejar de seguir',
+      watchAction: 'Seguir',
+      ownerTitle: 'PROPIETARIO',
+      contributorsTitle: 'CONTRIBUIDORES',
+      noContributorsMessage: 'No se encontraron contribuidores',
+      sourceTitle: 'CÓDIGO',
+      readMe: 'LEEME',
+      viewSource: 'Ver Código',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Ver código

File: src/locale/languages/es.js [hunk 26]
```diff
@@ -49,177 +51,183 @@ export const erase = {
...
-      noContributorsMessage: 'No contributors found',
-      sourceTitle: 'SOURCE',
-      readMe: 'README',
-      viewSource: 'View Code',
+      shareRepositoryTitle: 'Compartir {{repoName}}',
+      shareRepositoryMessage: 'Revisa {{repoName}} en GitHub. {{repoUrl}}',
+      repoActions: 'Acciones del repositorio',
+      forkAction: 'Hacer fork',
+      starAction: 'Marcar Favorito',
+      unstarAction: 'Desmarcar Favorito',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Desmarcar favorito

File: src/locale/languages/es.js [hunk 27]
```diff
@@ -49,177 +51,183 @@ export const erase = {
...
-      contributorsTitle: 'CONTRIBUTORS',
-      noContributorsMessage: 'No contributors found',
-      sourceTitle: 'SOURCE',
-      readMe: 'README',
-      viewSource: 'View Code',
+      shareRepositoryTitle: 'Compartir {{repoName}}',
+      shareRepositoryMessage: 'Revisa {{repoName}} en GitHub. {{repoUrl}}',
+      repoActions: 'Acciones del repositorio',
+      forkAction: 'Hacer fork',
+      starAction: 'Marcar Favorito',
```
Comment (ocarreterom) [inline src/locale/languages/es.js]: Marcar favorito

**Comment by khvilaboa** [comment]
@ocarreterom Done. Thanks!

**Comment by housseindjirdeh** [comment]
THANK YOU @khvilaboa @ocarreterom 

This is going to be amazing to have in 🇪🇸 🎉

**Pull request merged**